configure.ac

```diff
@@ -95,6 +95,22 @@ else
             [Defined if we're building with OpenSSL or LibreSSL])
 fi
 
+dnl Enable Android only features.
+AC_ARG_ENABLE(android,
+     AS_HELP_STRING(--enable-android, [setup build for Android exeutable/shared library]))
+
+if test "x$enable_android" = "xyes"; then
+  asciidoc=false
+  enable_html_manual=no
+  enable_manpage=no
+  enable_pic=yes
+  enable_system_torrc=no
+  enable_tool_name_check=no
+  have_systemd=no
+
+  LIBS="-llog $LIBS"
```

Comment:
Should the user have to build with "--enable-android" to get `-llog` ?  Everywhere else, we are using `__ANDROID__` to detect that library, so if the user forgets to say `--enable-android` they won't get it.

IMO we should either automatically include the logging library if building on android, or we should omit it when not using --enable-android.

Reply:
Makes sense, how do you want to do that?  I guess the `android/log.h` header check could be repurposed to do that.  My autotools is very rusty these days.

Reply:
I guess something like this would make more sense:
```
AC_CHECK_LIB([log], [__android_log_write], [LIBS="-llog $LIBS"])
```

Reply:
That would be great.